example/centos/centos-9-x86_64-vhd.yaml

```diff
@@ -0,0 +1,312 @@
+            - python3-pyyaml
+            - rpm
+            - selinux-policy-targeted
+            - systemd
+            - qemu-img
+            - xfsprogs
+            - xz
+          exclude: []
+    os:
+      otk.external.osbuild-gen-depsolve-dnf4:
+        architecture: x86_64
+        module_platform_id: c9s
+        releasever: "9"
+        repositories:
+          otk.include: "common/repositories/x86_64.yaml"
+        packages:
+          include:
+            - "@Server"
+            - NetworkManager
+            - NetworkManager-cloud-setup
+            - WALinuxAgent
+            - bzip2
+            - cloud-init
+            - cloud-utils-growpart
+            - dosfstools
+            - dracut-config-generic
+            - efibootmgr
+            - gdisk
+            - grub2-efi-x64
+            - grub2-pc
+            - hyperv-daemons
+            - kernel
+            - kernel-core
+            - kernel-modules
+            - langpacks-en
+            - lvm2
+            - nvme-cli
+            - patch
+            - rng-tools
+            - selinux-policy-targeted
+            - shim-x64
+            - uuid
+            - xfsprogs
+            - yum-utils
+          exclude:
+            - NetworkManager-config-server
+            - aic94xx-firmware
+            - alsa-firmware
+            - alsa-lib
+            - alsa-sof-firmware
+            - alsa-tools-firmware
+            - biosdevname
+            - bolt
+            - buildah
+            - cockpit-podman
+            - containernetworking-plugins
+            - dnf-plugin-spacewalk
+            - dracut-config-rescue
+            - glibc-all-langpacks
+            - iprutils
+            - ivtv-firmware
+            - iwl100-firmware
+            - iwl1000-firmware
+            - iwl105-firmware
+            - iwl135-firmware
+            - iwl2000-firmware
+            - iwl2030-firmware
+            - iwl3160-firmware
+            - iwl3945-firmware
+            - iwl4965-firmware
+            - iwl5000-firmware
+            - iwl5150-firmware
+            - iwl6000-firmware
+            - iwl6000g2a-firmware
+            - iwl6000g2b-firmware
+            - iwl6050-firmware
+            - iwl7260-firmware
+            - libertas-sd8686-firmware
+            - libertas-sd8787-firmware
+            - libertas-usb8388-firmware
+            - plymouth
+            - podman
+            - python3-dnf-plugin-spacewalk
+            - python3-hwdata
+            - python3-rhnlib
+            - rhn-check
+            - rhn-client-tools
+            - rhn-setup
+            - rhnlib
+            - rhnsd
+            - usb_modeswitch
+
+otk.include: "common/partition-table/x86_64/vhd_vmdk.yaml"
+
+otk.target.osbuild:
+  pipelines:
+    - otk.include: "pipeline/build/generic.yaml"
+    - name: os
+      build: name:build
+      stages:
+        - type: org.osbuild.kernel-cmdline
+          options:
+            root_fs_uuid: 9851898e-0b30-437d-8fad-51ec16c3697f
```

Comment:
Same comment as above about using a define here

Reply:
Done. I don't think it was available when this PR was initially made, it's in the added commit :)